service/azureconfig/v1/resource_set.go

```diff
@@ -180,3 +195,17 @@ func NewResourceSet(config ResourceSetConfig) (*framework.ResourceSet, error) {
 
 	return resourceSet, nil
 }
+
+func toCRUDResource(logger micrologger.Logger, ops framework.CRUDResourceOps) (*framework.CRUDResource, error) {
```

Comment:
Although I know it isn't everybodys favorite to do it like that it saves some lines, especially when there are many resources. This is private, so can be changed any time to a different init approach.

I'm open for suggestions.

Reply:
I would just keep it explicit.